src/System.Private.ServiceModel/src/System/ServiceModel/Description/ClientCredentials.cs

```diff
@@ -18,6 +18,7 @@ public class ClientCredentials : SecurityCredentialsManager, IEndpointBehavior
         private WindowsClientCredential _windows;
         private HttpDigestClientCredential _httpDigest;
         private bool _isReadOnly;
+        private bool _useIdentityConfiguration = false;
```

Comment:
ok

---
In reply to: [78485628](https://github.com/dotnet/wcf/pull/1515#discussion_r78485628) [](ancestors = 78485628)